src/components/organisms/ReceiverModal/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { useEffect } from 'react';
+import { useReceiverForm } from '@/hooks/useReceiverForm';
+import { ReceiverForm } from '@/components';
+import * as S from './styles';
+
+interface ReceiverModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  onCancel?: () => void;
+  onComplete?: () => void;
+}
+
+const ReceiverModal = ({
+  isOpen,
+  onClose,
+  onCancel,
+  onComplete,
+}: ReceiverModalProps) => {
+  const formHook = useReceiverForm();
+  const { addReceiver, canAddMore, fields, handleSubmit } = formHook;
+
+  useEffect(() => {
+    if (isOpen) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = 'unset';
+    }
+    return () => {
+      document.body.style.overflow = 'unset';
+    };
+  }, [isOpen]);
+
+  if (!isOpen) return null;
+
+  const handleCancel = () => {
+    onCancel?.();
+    onClose();
+  };
+
+  const handleComplete = () => {
+    handleSubmit(
+      () => {
+        onComplete?.();
+        onClose();
+      }
+    );
+  };
+
+  return (
+    <S.ModalOverlay>
+      <S.ModalContent>
+        <S.Header>
+          <S.Title>받는 사람</S.Title>
+          <S.Description>
+            * 최대 10명까지 추가 할 수 있어요.<br />
+            * 받는 사람의 전화번호를 중복으로 입력할 수 없어요.
+          </S.Description>
+          <S.AddButton 
+            onClick={addReceiver}
+            disabled={!canAddMore}
+          >
+            추가하기
+          </S.AddButton>
+        </S.Header>
+        <S.ContentArea>
+          <ReceiverForm formHook={formHook} />
+        </S.ContentArea>
+        <S.ButtonArea>
+          <S.CancelButton onClick={handleCancel}>
+            취소
+          </S.CancelButton>
+          <S.CompleteButton onClick={handleComplete}>
```

Comment:
```suggestion
          <S.CompleteButton onClick={handleSubmit}>
```

요렇게 바로 넣으면 안 되나요..? 한 번 함수로 감싸신 이유가 있나 해서..

handleCancel 도요!

Reply:
처음 만들 때 아 이런 거 있어야 돼, 저런 거 있어야 돼 하면서 이거저거 다 만들다 보니 남아있는 것들이 있었던 것 같습니다. 간단하게 변경했습니다.